pkg/gui/gui.go

```diff
@@ -49,6 +49,9 @@ func (gui *Gui) GenerateSentinelErrors() {
 	}
 }
 
+// Teml is short for template used to make the required map[string]interface{} shorter when using gui.Tr.SLocalize and gui.Tr.TemplateLocalize
```

Comment:
can we put this in the i18n package and then call it as i18n.Temp, just incase we need to call it from another package as well